web_external/js/models/DatasetModel.js

```diff
@@ -91,7 +99,7 @@ minerva.models.DatasetModel = minerva.models.MinervaModel.extend({
                     type: 'wms'
                 };
             } else {
-                console.error('Unknown dataset_type for geo_render purposes: ' + mm.dataset_type);
+                console.log('Unknown dataset_type for geo_render purposes: ' + mm.dataset_type);
```

Comment:
It's fine to have a csv file that can display as a csv, but currently there is no mechanism to get this displayed on the map.  We don't display the eyeball to display it on the map, so there isn't a problem of trying to render something that can't be rendered on the map.

Hence, it's not an error, but it might be something to be aware of.  Probably it could be removed altogether, but at this stage I'm erring on the side of letting developers know what is going on.